[Test:] `azurerm_cdn_profile`/`azurerm_cdn_endpoint`/`azurerm_frontdoor` - fix tests due to deprecated `sku` values by RP (#28908)
hashicorp/terraform-provider-azurerm: website/docs/r/cdn_endpoint_custom_domain.html.markdown

Manages a Custom Domain for a CDN Endpoint.

> !> **Note:** The CDN services from Edgio(formerly Verizon) will be shut down on or before 15 January 2025. All Azure CDN from Edgio customers must migrate their workloads out of Azure CDN from Edgio before 15 January 2025.
> 
> !> **Note:** The CDN services from Akamai support will be removed on 31 October 2023. All Azure CDN from Akamai customers must migrate their workloads out of Azure CDN from Akamai before 31 October 2023.

Reviewer: As above?
```suggestion
!> **Note:** The CDN services from Edgio(formerly Verizon) was shut down on 15 January 2025 and is no longer available .

!> **Note:** Support for CDN services from Akamai was removed on 31 October 2023. 
```
